packages/contracts/scripts/shared/types/handler.ts

```diff
@@ -13,3 +14,13 @@ export type TaskFuncParam = {
 };
 
 export type TaskFuncCallBack = (params: TaskFuncParam) => Promise<any>;
+
+export type ArgsType = {
```

Comment:
Lets rename `ArgsType` to something else so anyone can guess its meaning.  I don't have the best name right now but at least we need to remove `Type` from `ArgsType`.

Here are the suggested names: `Arguments`, `InputParam`... You can think of a better name as well.